### DISCUSSION TITLE
Error: nuxt instance unavailable when catch error with pinia

### DISCUSSION THREAD

**Discussion by MrJmpl3** [nuxt/nuxt#17147] [Questions]
When I tried to catch a error of actions in pinia, I got "nuxt instance unavailable"

https://stackblitz.com/edit/github-v7qjzm-xymukj?file=stores/counter.ts

I don't know if a problem of me, Nuxt or Pinia.

I look Pinia docs and the examples used: `this.$nuxt`

**Comment by Lanayx** [comment]
+1 to this. [Release notes](https://github.com/vuejs/pinia/blob/v2/packages/nuxt/CHANGELOG.md#030-2022-07-13) say  
`nuxt: in Nuxt 3, $nuxt is no longer available in stores. This is because it was removed in Nuxt 3 and it is no longer the context as it used to be. Most of the features used there, like $fetch are now globally available and therefore remove the need of it. You can also use [useNuxtApp()](https://v3.nuxtjs.org/bridge/bridge-composition-api/) when necessary.`
However useNuxtApp() is not available in store, it gives `Error: nuxt instance unavailable` Can we have it returned back
